web/public/locales/no-NB.json

```
@@ -705,7 +752,7 @@
       "zoneName": "Guyana"
     },
     "HK": {
-      "zoneName": "Hong Kong"
+      "zoneName": "Hong-Kong"
```

```suggestion
      "zoneName": "Hongkong"
```

At least according to Wikipedia 😅

---

Seems like this was not changed, did you perhaps forget to push it or was the dashed version actually correct? :)